main.rb

```diff
@@ -0,0 +1,168 @@
+require_relative 'lib/Book/book_manager'
+require_relative 'lib/Music/music_album_manager'
+require_relative 'lib/Game/Managers/game_manager'
+require_relative 'lib/Data/store_data'
+require_relative 'lib/Data/retrieve_data'
+
+def display_header
+  puts "\e[35m" # Set the text color to purple
+  puts '***********************************************************'
+  puts '  📚 🎶 🎮  Welcome to My-Catalog Console App  🎮 🎶 📚  '
+  puts '***********************************************************'
+  puts ''
+  puts '🌟 Get ready to 🌍 explore, 🕵️‍♂️ discover, and 🚀 have a blast! 🎉'
+  puts ''
+  puts '🤔 What would you like to do today? 🤖'
+  puts "\e[34m"
+end
+
+ACTIONS = {
+  1 => { method: :list_books },
+  2 => { method: :list_music_albums },
+  3 => { method: :list_games },
+  4 => { method: :list_genres },
+  5 => { method: :list_labels },
+  6 => { method: :list_authors },
+  7 => { method: :add_book },
+  8 => { method: :add_music_album },
+  9 => { method: :add_game },
+  10 => { method: :remove_music_album },
+  11 => { method: :exit_app }
+}.freeze
+
+class Main
+  def initialize
+    @book_manager = BookManager.new
+    @music_album_manager = MusicAlbumManager.new
+    @game_manager = GameManager.new
+    @store_data = StoreData.new
+    @retrieve_data = RetrieveData.new
+  end
+
+  def run
+    @retrieve_data.retrieve_all(@game_manager, @music_album_manager, @book_manager)
+    selected_option = nil
+    loop do
+      system('clear')
+      display_options(selected_option)
+      number = gets.chomp.to_i
+      selected_option = number
+      if number.zero?
+        @store_data.store_all(@game_manager, @music_album_manager, @book_manager)
+        exit_msg
+        break
+      end
+      action = ACTIONS[number]
+      if action
+        send(action[:method])
+      else
+        puts 'Enter a valid number'
+      end
+      puts ''
+    end
+  end
+
+  def display_options(selected_option)
+    options = [
+      '1️⃣   List all books 📚',
+      '2️⃣   List all music albums 🎶 📀',
+      '3️⃣   List of games 🎮',
+      '4️⃣   List all genres  🎵 🎶',
+      '5️⃣   List all labels 🏷️  📚',
+      '6️⃣   List all authors ✍️  📖',
```

Comment:
- [ ] As required for the project your application to work correctly, kindly ensure that the selected options above are working as expected. Right now, while having books, music_albums, games,... in your data base, nothing is working with selected options above